plugins/analysis-icu/build.gradle

```diff
@@ -18,17 +18,20 @@ import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApis
  * specific language governing permissions and limitations
  * under the License.
  */
-apply plugin: 'elasticsearch.rest-resources'
+apply plugin: 'elasticsearch.yaml-rest-test'
+apply plugin: 'elasticsearch.internal-cluster-test'
 
 esplugin {
   description 'The ICU Analysis plugin integrates the Lucene ICU module into Elasticsearch, adding ICU-related analysis components.'
   classname 'org.elasticsearch.plugin.analysis.icu.AnalysisICUPlugin'
 }
 
-tasks.withType(CheckForbiddenApis).configureEach {
-  signatures += [
-    "com.ibm.icu.text.Collator#getInstance() @ Don't use default locale, use getInstance(ULocale) instead"
-  ]
+List<String> forbidden_signatures = new ArrayList<>([
```

Comment:
What's the purpose of this of this intermediary variable?

Reply:
it made sense when i was configuring multiple forbidden[name] ... but now not so much...will fix.

Reply:
done https://github.com/elastic/elasticsearch/pull/59444/commits/849d19db73f47c21baaa3cb591d28521bd9e6294